DMCompiler/DMStandard/Types/Datum.dm

```diff
@@ -4,8 +4,12 @@
 
 	var/list/vars
 
-	var/tag = null as opendream_unimplemented
+	var/tag
 
 	proc/New()
+
 	proc/Del()
-	proc/Topic(href, href_list)
\ No newline at end of file
+		if(tag)
+			tag = null
```

Comment:
when it's not 5am I need to check and make sure the object is actually being removed from the queue when deleted

Reply:
Seems fine